dvc/ui/table.py

```diff
@@ -14,7 +14,7 @@
 SHOW_MAX_WIDTH = 1024
 
 
-CellT = Union[str, "RichText"]  # RichText is mostly compatible with str
+CellT = Union[str, "RichText", None]  # RichText is mostly compatible with str
```

Comment:
Please don't extend the types here. Table rendering does not support `None` for now.

Reply:
Then what should I set here ?
https://github.com/iterative/dvc/blob/eac030598cdbfaba4c029f8e1131614916a6c707/dvc/command/machine.py#L253
It looks like an empty string didn't work in this case.

Reply:
Use a fill value "-" or just empty string `""`.

Reply:
Fill value is a bit too hard code I think, and empty string `""` it was just because it gives empty result instead of a fill value made me using None in this case.

Reply:
@skshetry How about adding a new type `InputCellT` which can accept `None` value for those input functions like `append`, `extend` or `row_from_dict`?

Reply:
@karajan1001, we'll need to fix that in https://github.com/iterative/dvc/issues/7167. Meanwhile, this may introduce unnecessary bugs, as rendering with `rich` table is not capable of supporting `None`, could you please roll back the type of `CellT`, and make adjustments, please?

Reply:
> @karajan1001, we'll need to fix that in #7167. Meanwhile, this may introduce unnecessary bugs, as rendering with `rich` table is not capable of supporting `None`, could you please roll back the type of `CellT`, and make adjustments, please?

Sure.